dev/conductor/core/lib/src/codesign.dart

```diff
@@ -73,9 +73,8 @@ class CodesignCommand extends Command<void> {
 
   FrameworkRepository? _framework;
   FrameworkRepository get framework {
-    return _framework ??= FrameworkRepository.localRepoAsUpstream(
+    return _framework ??= FrameworkRepository(
```

Comment:
Is the engine already checking out from remote?

Reply:
We don't clone the engine as part of this flow. We get the binaries from exercising the flutter tool's caching mechanism.